src/CHANGELOG.js

```diff
@@ -2,7 +2,6 @@ export default `
 20-08-2017 - Restoration Shaman: Added Ancestral Vigor metric. (by Yuyz0112)
 19-08-2017 - Guardian Druid: Added detail on Ironfur usage.
 19-08-2017 - Vengeance Demon Hunter: Added more useful and specific statistics tooltips.
-18-08-2017 - Added Affliction Warlock support by @Chizu.
```

Comment:
Don't really know, I updated my fork ("Merge remote-tracking branch 'upstream/master'") and when I looked through what was changed I noticed there is another line with Added Affliction Warlock support. Don't know where that came from but I liked the colored version more :D